README.md

```diff
@@ -1 +1,55 @@
-# React-exercise-Todo-List
\ No newline at end of file
+# Todo List - React
```

Comment:
![giphy (2)](https://user-images.githubusercontent.com/93735137/154510514-c926170e-197d-43cb-8b51-3169bd1cb196.gif)
Hi @VuDej,

Good job so far!

 The app is full operative and made under tutorial with classes and hooks.
 There are some issues that you still need to work on to go to the next project but you are almost there!
Required Changes recycle
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

N/A
Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.